modules/ctransformers_model.py

```diff
@@ -9,29 +12,57 @@ class CtransformersModel:
     def __init__(self):
         pass
 
-    @classmethod
```

Comment:
Why remove the classmethod? Is there an advantage?

Reply:
It's for consistency with the other methods that use self. The classmethod wasn't intended.